diff --git a/node_modules/react-native-deck-swiper/Swiper.js b/node_modules/react-native-deck-swiper/Swiper.js
index 7cbb4a9..6d52e22 100644
--- a/node_modules/react-native-deck-swiper/Swiper.js
+++ b/node_modules/react-native-deck-swiper/Swiper.js
@@ -1,4 +1,5 @@
 import React from "react";
+import PropTypes from "prop-types";
 import {
   PanResponder,
   Easing,
@@ -415,39 +416,39 @@ class Swiper extends React.Component {
 }
 
 Swiper.propTypes = {
-  cards: React.PropTypes.array.isRequired,
-  childrenOnTop: React.PropTypes.bool,
-  renderCard: React.PropTypes.func.isRequired,
-  onSwipedAll: React.PropTypes.func,
-  onSwiped: React.PropTypes.func,
-  onSwipedLeft: React.PropTypes.func,
-  onSwipedRight: React.PropTypes.func,
-  onSwipedTop: React.PropTypes.func,
-  onSwipedBottom: React.PropTypes.func,
-  cardIndex: React.PropTypes.number,
-  infinite: React.PropTypes.bool,
-  secondCardZoom: React.PropTypes.number,
-  zoomFriction: React.PropTypes.number,
-  backgroundColor: React.PropTypes.string,
-  marginTop: React.PropTypes.number,
-  marginBottom: React.PropTypes.number,
-  cardVerticalMargin: React.PropTypes.number,
-  cardHorizontalMargin: React.PropTypes.number,
-  outputRotationRange: React.PropTypes.array,
-  inputRotationRange: React.PropTypes.array,
-  animateOpacity: React.PropTypes.bool,
-  inputOpacityRange: React.PropTypes.array,
-  outputOpacityRange: React.PropTypes.array,
-  verticalThreshold: React.PropTypes.number,
-  horizontalThreshold: React.PropTypes.number,
-  previousCardInitialPositionX: React.PropTypes.number,
-  previousCardInitialPositionY: React.PropTypes.number,
-  swipeAnimationDuration: React.PropTypes.number,
-  swipeBackAnimationDuration: React.PropTypes.number,
-  zoomAnimationDuration: React.PropTypes.number,
-  swipeBackFriction: React.PropTypes.number,
-  horizontalSwipe: React.PropTypes.bool,
-  verticalSwipe: React.PropTypes.bool
+  cards: PropTypes.array.isRequired,
+  childrenOnTop: PropTypes.bool,
+  renderCard: PropTypes.func.isRequired,
+  onSwipedAll: PropTypes.func,
+  onSwiped: PropTypes.func,
+  onSwipedLeft: PropTypes.func,
+  onSwipedRight: PropTypes.func,
+  onSwipedTop: PropTypes.func,
+  onSwipedBottom: PropTypes.func,
+  cardIndex: PropTypes.number,
+  infinite: PropTypes.bool,
+  secondCardZoom: PropTypes.number,
+  zoomFriction: PropTypes.number,
+  backgroundColor: PropTypes.string,
+  marginTop: PropTypes.number,
+  marginBottom: PropTypes.number,
+  cardVerticalMargin: PropTypes.number,
+  cardHorizontalMargin: PropTypes.number,
+  outputRotationRange: PropTypes.array,
+  inputRotationRange: PropTypes.array,
+  animateOpacity: PropTypes.bool,
+  inputOpacityRange: PropTypes.array,
+  outputOpacityRange: PropTypes.array,
+  verticalThreshold: PropTypes.number,
+  horizontalThreshold: PropTypes.number,
+  previousCardInitialPositionX: PropTypes.number,
+  previousCardInitialPositionY: PropTypes.number,
+  swipeAnimationDuration: PropTypes.number,
+  swipeBackAnimationDuration: PropTypes.number,
+  zoomAnimationDuration: PropTypes.number,
+  swipeBackFriction: PropTypes.number,
+  horizontalSwipe: PropTypes.bool,
+  verticalSwipe: PropTypes.bool
 };
 
 Swiper.defaultProps = {
